pkg/universe.dagger.io/alpha/codecov/upload.cue

```diff
@@ -0,0 +1,52 @@
+package codecov
+
+import (
+	"dagger.io/dagger"
+
+	"universe.dagger.io/docker"
+)
+
+#Upload: {
+	// Source code
+	source: dagger.#FS
+
+	// Coverage files
+	file: string
```

Comment:
Maybe rename this to files or accept a list as well and concatenate them when necessary.